ansible/monitoring.yml

```diff
@@ -88,6 +92,10 @@
         init_config:
         instances:
         logs:
+          - type: tcp
+            port: 10518
+            service: epoch
```

Comment:
I understand this [`service`](https://docs.datadoghq.com/logs/log_collection/?tab=tailexistingfiles) is just a label that datadog **adds** to the data it reads from port. I.e. it is not a filter on which lines of the syslog are meant to be sent. And has no connection whatsoever with Unix process names (or with which command a process was launched).

That leads to the question: does this mean that anything that goes in syslog is sent to datadog? (I see no filtering in ansible/files/rsyslog/datadog.conf )

And if yes, all with tag `epoch`? (Not sure if this will cause confusion in the UI - as if logs were from app.)

And is the severity filtered at all? I see nothing in "ansible/files/rsyslog/datadog.conf" and I do not understand `log_level`.

Reply:
Yes, `service: epoch` is for filtering purposes (in the UI). We have other services too (not yet feeding datadog logs). This is further narrowed by `source: syslog` to distinguish from epoch daemon logs itself. I suppose we can choose better name for the service in future, this was already used.

Anything that goes to syslog is send to the logs collector, we might narrow the scope once we have more statistics in few days. My smoke tests doesn't show a lot of records.

Yes, all the records are "tagged" as `service: epoch` and `source: syslog`

The `log_level` above is not related to the syslog configuration, it only lower the datadog agent severity.